CMakeLists.txt

```diff
@@ -122,6 +122,10 @@ set_package_properties(MKL PROPERTIES TYPE OPTIONAL PURPOSE "Enables accelerated
 if(MKL_FOUND AND CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,--no-as-needed")
 endif()
+option(REQUIRE_MKL "Require MKL" OFF)
```

Comment:
@JensWehner that and https://github.com/votca/actions/commit/99eead7cf2820d7666d5885b89b78e4c7a4e6763 should fix the silent failure with mkl.